.env.placeholder

```
@@ -5,7 +5,7 @@ VITE_IDP_SCOPE=VITE_IDP_SCOPE_PLACEHOLDER
 VITE_IDP_RESOURCE=VITE_IDP_RESOURCE_PLACEHOLDER
 VITE_IDP_POST_LOGOUT_REDIRECT_PATH=VITE_IDP_POST_LOGOUT_REDIRECT_PATH_PLACEHOLDER
 VITE_IDP_TOKEN_TYPE=VITE_IDP_TOKEN_TYPE_PLACEHOLDER
-VITE_ENABLE_AUTHENTICATION=VITE_ENABLE_AUTHENTICATION_PLACEHOLDER
-VITE_ENABLE_PERMISSIONS=VITE_ENABLE_PERMISSIONS_PLACEHOLDER
+VITE_ENABLE_AUTHENTICATION=true
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_ENABLE_AUTHENTICATION key should go before the VITE_IDP_AUTHORITY key

(UnorderedKey)